presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -1044,4 +1047,25 @@ public boolean isTableStatisticsEnabled()
     {
         return tableStatisticsEnabled;
     }
+
+    @NotNull
+    public CollectColumnStatisticsOnWriteOption getCollectColumnStatisticsOnWrite()
+    {
+        return collectColumnStatisticsOnWrite;
+    }
+
+    @Config("hive.collect-column-statistics-on-write")
+    @ConfigDescription("Enables automatic column level statistics collection on write")
+    public HiveClientConfig setCollectColumnStatisticsOnWrite(CollectColumnStatisticsOnWriteOption collectColumnStatisticsOnWrite)
+    {
+        this.collectColumnStatisticsOnWrite = collectColumnStatisticsOnWrite;
+        return this;
+    }
+
+    public enum CollectColumnStatisticsOnWriteOption
+    {
+        ENABLED,
+        ENABLED_FOR_MARKED_TABLES,
```

Comment:
add a javadoc linking to `com.facebook.presto.hive.HiveTableProperties#COLLECT_COLUMN_STATISTICS_ON_WRITE_ENABLED`

Reply:
I removed the enum